src/backlight/backlight.c

```diff
@@ -338,8 +338,9 @@ static int clamp_brightness(
          * avoids preserving an unreadably dim screen, which would otherwise force the user to disable
          * state restoration. */
 
+        min_brightness = (unsigned) ((double) max_brightness * percent / 100);
         if (device_in_subsystem(device, "backlight"))
-                min_brightness = MAX(1U, (unsigned) ((double) max_brightness * percent / 100));
+                min_brightness = MAX(1U, min_brightness);
         else
                 min_brightness = 0;
```

Comment:
This should be removed? The minimum percentage is handled in shall_clamp now.

Reply:
Ouch. Fixed. Thanks.